samples/snippets/src/main/java/com/google/cloud/gameservices/samples/realms/CreateRealm.java

```
@@ -0,0 +1,63 @@
+
+// [START cloud_game_servers_realm_create]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.gaming.v1alpha.CreateRealmRequest;
+import com.google.cloud.gaming.v1alpha.Realm;
+import com.google.cloud.gaming.v1alpha.RealmsServiceClient;
+import com.google.protobuf.Empty;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateRealm {
+  public static void createRealm(String projectId, String regionId, String realmId) {
+    // String projectId = "your-project-id";
+    // String regionId = "us-central1-f";
+    // String realmId = "your-realm-id";
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (RealmsServiceClient client = RealmsServiceClient.create()) {
+      String parent = String.format("projects/%s/locations/%s", projectId, regionId);
+      String realmName = String.format("%s/realms/%s", parent, realmId);
+
+      Realm realm =
+          Realm.newBuilder().setName(realmName).setTimeZone("America/Los_Angeles").build();
+
+      CreateRealmRequest request =
+          CreateRealmRequest.newBuilder()
+              .setParent(parent)
+              .setRealmId(realmId)
+              .setRealm(realm)
+              .build();
+
+      OperationFuture<Realm, Empty> call = client.createRealmAsync(request);
+      Realm result = call.get(1, TimeUnit.MINUTES);
+
+      System.out.println("Realm created: " + result.getName());
+    } catch (IOException | InterruptedException | ExecutionException | TimeoutException e) {
+      System.err.println("Realm create request unsuccessful.");
+      e.printStackTrace(System.err);
+    }
```

Why catch this rather than just let it propagate?
If you catch something, at the least, you should suggest what might of happened and what the user might do to mitigate the problem.